packages/logger/src/lib/create-logger.ts

```diff
@@ -0,0 +1,77 @@
+import { LogConsumer } from './model/log-consumer';
+import { Logger } from './model/logger';
+import { LogLevel } from './model/log-level';
+import { getGlobalLogLevel } from './global-log-level';
+
+export function createLogger(): Logger {
```

Comment:
Not a fan that there can be multiple "root" logger instances.

This can get really confusing if you are not using some DI to inject the Logger everywhere, because suddenly your logger does not print anything, since you did not register any consumers.

Reply:
Done this way by design for customizability. There can be cases where one wants to separate consumers. Think about some big UI fragments from somewhere else loaded into the app.